spec/models/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
do we need type?

Reply:
I thought it was good practice bhaiya. Should I remove it?

Reply:
yes

Reply:
removed it bhaiya